### DISCUSSION TITLE
I have a question about the package requirements

### DISCUSSION THREAD

**Discussion by Boria138** [void-linux/void-packages#49577] [Q&A]
A person wanted to add the PortProton package https://github.com/void-linux/void-packages/pull/49328, but in the end the request was rejected because the package is not built, the thing is that this package does not need to be built, it is an installation script that loads all the necessary files at startup, so I want to ask how to add such a package ? If I for example add meson as a build system and use it to add files where needed, will it be considered a build or not ?

**Comment by iFoundSilentHouse** [comment]
Well because it's an install script that installs PortProton prefix on chosen folder and <b>not</b>  system-wide, it doesn't apply to any package requirements. I was mistakenly claiming it ok with package requirements and didn't provide any arguments why portproton is an exception to be packaged surpassing rules. That's why PR was just closed.

Anyways. I PRd void linux dependencies to PortProton script page https://github.com/Castro-Fidel/PortWINE you can just use commands from the bottom of page and use install script from official page ;)
